tests/storage/storage.go

```diff
@@ -259,8 +259,12 @@ var _ = SIGDescribe("Storage", func() {
 					libwait.WaitForVirtualMachineToDisappearWithTimeout(vmi, 120)
 
 					if targetImagePath != testsuite.HostPathAlpine {
-						tests.DeleteAlpineWithNonQEMUPermissions()
-					}
+						// delete the Alpine image with non-QEMU permissions
```

Comment:
Thanks for cleaning up tests/utils.go. This is a noble cause.

However, if you feel that a comment is needed to make a piece of code more readable, it would be better to keep the code in a function. This should be the case here.

Also, `DeleteAlpineWithNonQEMUPermissions` cleans up resources that were created by `CopyAlpineWithNonQEMUPermissions`. Please keep both functions next to each other.

Reply:
Hi @sarthaksarthak9 thanks for this pr! are you planing on working on this more?

Reply:
yes